docker/yb-voyager-docker

```diff
@@ -3,24 +3,48 @@
 argv=( "$@" )
 
 if ! which docker > /dev/null; then
-   echo "Could not find docker! Please install docker before proceeeding."
-   exit 1
+   	echo "Could not find docker! Please install docker before proceeeding."
+   	exit 1
 fi
 
-if [[ $OSTYPE == 'darwin'* ]]; then
-  platform="--platform=linux/amd64"
+if [[ $OSTYPE == 'darwin'* ]]
+then
+	platform="--platform=linux/amd64"
 fi
 
 i=0
 while [ $i -lt $# ]
 do
-    if [[ ${argv[${i}]} == "--export-dir" || ${argv[${i}]} == "-e" ]]
-    then
-        j=$(( $i + 1))
-        vol="-v ${argv[$j]}:/export-dir"
-        argv[$j]='/export-dir'
-    fi
-    i=$(( $i + 1))
+    	if [[ ${argv[${i}]} == "--export-dir" || ${argv[${i}]} == "-e" ]]
+    	then
+        	j=$(( $i + 1))
+		
+			# If the OS is macOS
+			if [[ $OSTYPE == 'darwin'* ]]; then
+
+				# Check if a volume voyager-export-dir binded to the export dir does not exist			
+				bindedExportDir=$(docker inspect voyager-export-dir | grep "device" | sed -n 's/"device": *"\([^"]*\).*/\1/p' | tr -d " ")
```

Comment:
For local variables, use all lower case names separated by underscores. e.g. `bound_export_dir`.

`bound_export_dir=$(docker volume inspect voyager-export-dir -f '{{.Mountpoint}}')`

Reply:
Removed this line completely because of the other change below.